README.md

```diff
@@ -71,7 +71,39 @@ Make sure you have `nvm` and `docker` (rootless) installed, then run
 ./local-dev.sh
 ```
 
-This will set everything up with reasonable defaults for a local dev.
+This will set the project up with reasonable defaults for a local dev.
+
+To run the ReefGuide webapp, environment files for the `web-api` and `cli` need to be
```

Comment:
I think this should be moved into the manual setup. The local dev script sets things up AND runs them - if this is not working, we should fix the script. The web-api .env file is definitely copied, the cli one is not as the local dev does not consider the CLI as in scope for setup (but it could, we can add that). 

The instructions are good, but should mostly be integrated into the manual dev setup section.

The instructions to setup the other ReefGuide repositories are helpful in both sections - so maybe split that off as a "setup worker nodes for local development" ? I usually setup local dev first, then setup my workers.

Reply:
Thanks @PeterBaker0

Re the CLI setup, I guess it's a matter of semantics. By `local-dev`, I interpreted as meaning "set everything to do with reefguide-webapp development", whereas it means "set the webapp up".

If it's a small change to the setup script, I would like the CLI setup to be included 😄 

I think I'm missing a small direction to readers about where to look for or setup login credentials, but to your point this is a separate section.

Will make your suggested adjustments soon.